omp/test/components/fill_array.cpp

```diff
@@ -74,8 +74,7 @@ class FillArray : public ::testing::Test {
     gko::Array<value_type> seqs;
 };
 
-TYPED_TEST_SUITE(FillArray, gko::test::ValueAndIndexTypes,
-                 TypenameNameGenerator);
+TYPED_TEST_SUITE(FillArray, gko::test::AllTemplateTypes, TypenameNameGenerator);
```

Comment:
this change seems left over from the unsigned index approach.

Reply:
`ValueAndIndexTypes` still instantiates for `size_type`. Should we keep that ?